### ISSUE TITLE
unlock PHP 7.1+

### ISSUE THREAD

**Issue by DanielRuf** [nuxsmin/sysPass#731]
Currently sysPass is locked to PHP 5.6 - 7.0. It would be good to allow new versions.
We should create an extra branch for this, add Travis CI and test it for possible issues.

**Comment by nuxsmin** [comment]
You're right, it mostly should run on PHP 7.1 but I've found some minor bugs that need to be fixed.

Besides these bugs, I'd like to implement PHP 7.x features like type checking.

Next version will be fully compatible.

Thanks for the feedback!

**Comment by DanielRuf** [comment]
In some cases https://github.com/symfony/polyfill might be useful.

**Comment by ejouvin** [comment]
+1
I have just upgrade my Ubuntu server to version 17.10 and phph version is now 7.1
My instance can not work now.
So waiting for the fix ;)

Good luck

**Comment by nuxsmin** [comment]
;)

**Comment by ejouvin** [comment]
By the way, do you have any expected date for a 7.1 php fix ?

(Why did I upgrade ubuntu also)

**Comment by nuxsmin** [comment]
Hello, I'll try out to get it for the next release, but if there would have major changes, it would be delayed until next major release.

Kind regards
